src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unolib-crossruntime/.template.config/template.json

```diff
@@ -43,9 +43,13 @@
       "path": "UnoCrossRuntimeLib\\UnoCrossRuntimeLib.csproj"
     },
     {
-      "condition": "Android",
+      "condition": "WebAssembly",
```

Comment:
@jeromelaban Do we use "Wasm" or "WebAssembly" here?

Reply:
`WebAssembly` is fine. The shortname may be wasm:

https://github.com/unoplatform/uno/blob/540399456be9e6caf7d7616fdc438ee7ea81cb89/src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp-net6/.template.config/dotnetcli.host.json#L4-L7

It's fine like this.